### DISCUSSION TITLE
How do you switch between applications?

### DISCUSSION THREAD

**Discussion by ninjarogue** [regolith-linux/regolith-desktop#708] [Q&A] [answered]
Is there a better way of switching between applications (not including `SUPER` + `SPACE`). If the applications are not full-screen I can do a `SUPER` + left or right arrow to switch between my apps but I want to be able to switch between apps in full-screen mode. Is this possible?

**Comment by kgilmer** [comment, marked as answer]
Hi @ninjarogue , hmm, I cannot say that what you ask is impossible, but I would do it a bit differently.  I would put each full-screen application on it's own workspace and switch between them with `<super>` `<number>`.  Or `<super>``<tab>` / `<super>``<shift>``<tab>` for next / previous workspace.  You can also use the window navigation dialog with `<super>``<control>``<space>` if you have 3 or more apps you want to switch between.
